CodeSniffer/Standards/Generic/Docs/Functions/ReturnTypeDeclarationStandard.xml

```diff
@@ -0,0 +1,28 @@
+<documentation title="Return Type Declaration">
+    <standard>
+        <![CDATA[
+        Return type declarations are optional, and have a specified amount of spaces between the function's closing parenthesis and colon, and a
+        specified amount of spaces between the colon and the return type.
+        The default is zero spaces between closing parenthesis and colon and one space between colon and return type.
+        The return type should be lowercase.
```

Comment:
You may want to update this line in the description to reflect that class names don't have to be lowercase.